spec/payloads/api/current/patient_payload_validator_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+def new_patient_payload(attrs = {})
+  payload = Api::Current::PatientPayloadValidator.new(build_patient_payload.deep_merge(attrs))
+  payload.validate
+  payload
+end
+
+describe Api::Current::PatientPayloadValidator, type: :model do
+  describe "Validations" do
+    it "Validates that either age or date of birth is present" do
+      expect(new_patient_payload('address'       => nil,
+                                 'phone_numbers' => nil,
+                                 'age'           => nil,
+                                 'date_of_birth' => Date.today).valid?).to be true
+
+      expect(new_patient_payload('address'        => nil,
+                                 'phone_numbers'  => nil,
+                                 'age'            => rand(18..100),
+                                 'age_updated_at' => Time.now,
+                                 'date_of_birth'  => nil).valid?).to be true
+
+      expect(new_patient_payload('address'        => nil,
+                                 'phone_numbers'  => nil,
+                                 'age'            => rand(18..100),
+                                 'age_updated_at' => nil,
+                                 'date_of_birth'  => nil).valid?).to be false
+
+      expect(new_patient_payload('address'       => nil,
+                                 'phone_numbers' => nil,
+                                 'age'           => nil,
+                                 'date_of_birth' => nil).valid?).to be false
+    end
+
+    it 'validates patient date_of_birth is less than today' do
+      payload = new_patient_payload('date_of_birth' => 3.days.from_now)
+      expect(payload.valid?).to be false
+      expect(payload.errors[:date_of_birth]).to be_present
+    end
+
+    it "Validates json spec for patient sync request" do
```

Comment:
Split these into separate `it` blocks?

Reply:
This change seems unrelated to this PR. Don't know why it shows up here. I don't remember making it. Will check.

Reply:
Ah, I see I versioned the spec file, because the v1 test was different. Fixed in 11a00c59fe892d3b6e893c5961ce7e49e3617f7e.